securedrop_client/gui/widgets.py

```diff
@@ -1588,44 +1619,109 @@ def __init__(
         Given some text and a reference to the controller, make something to display a file.
         """
         super().__init__()
+
         self.controller = controller
         self.file = self.controller.get_file(file_uuid)
 
-        self.layout = QHBoxLayout()
-        self.update()
-        self.setLayout(self.layout)
+        # Set styles
+        self.setObjectName('file_widget')
+        self.setStyleSheet(self.CSS)
+        file_description_font = QFont()
+        file_description_font.setLetterSpacing(QFont.AbsoluteSpacing, self.FILE_FONT_SPACING)
+        file_buttons_font = QFont()
+        file_buttons_font.setLetterSpacing(QFont.AbsoluteSpacing, self.FILE_OPTOINS_FONT_SPACING)
 
-        file_ready_signal.connect(self._on_file_downloaded, type=Qt.QueuedConnection)
+        # Set layout
+        layout = QHBoxLayout()
+        self.setLayout(layout)
 
-    def update(self) -> None:
-        icon = QLabel()
-        icon.setPixmap(load_image('file.png'))
+        # Set margins and spacing
+        layout.setContentsMargins(0, self.TOP_MARGIN, 0, 0)
+        layout.setSpacing(0)
+        self.setSizePolicy(QSizePolicy.Fixed, QSizePolicy.Fixed)
 
+        # File options: download, export, print
+        self.file_options = QWidget()
+        file_options_layout = QHBoxLayout()
+        self.file_options.setLayout(file_options_layout)
+        self.download_button = QPushButton(_(' DOWNLOAD'))
+        self.download_button.setObjectName('download_button')
+        self.download_button.setIcon(load_icon('download_file.svg'))
+        self.download_button.setFont(file_buttons_font)
+        self.export_button = QPushButton(_('EXPORT'))
+        self.export_button.setObjectName('export_print')
+        self.export_button.setFont(file_buttons_font)
+        self.print_button = QPushButton(_('PRINT'))
+        self.print_button.setObjectName('export_print')
+        self.print_button.setFont(file_buttons_font)
+        file_options_layout.addWidget(self.download_button)
+        file_options_layout.addWidget(self.export_button)
+        file_options_layout.addWidget(self.print_button)
+
+        # File name or default string
+        self.file_name = SecureQLabel(self.file.original_filename)
+        self.file_name.setObjectName('file-name')
+        self.no_file_name = QLabel('ENCRYPTED FILE ON SERVER')
```

Comment:
another nit but thought for the future: we may as well use `SecureQLabel` basically everywhere `QLabel` is used, the reason being that right now this string is hardcoded by the developer but when we localize these strings, a translator could attempt to inject HTML here